app/src/main/java/org/sagebionetworks/research/mpower/viewmodel/StudyBurstSettingsDao.kt

```diff
@@ -0,0 +1,35 @@
+package org.sagebionetworks.research.mpower.viewmodel
+
+import android.content.Context
+import android.support.annotation.VisibleForTesting
+import com.google.gson.reflect.TypeToken
+import org.sagebionetworks.bridge.rest.RestUtils
+import org.threeten.bp.LocalDateTime
+import javax.inject.Inject
+
+open class StudyBurstSettingsDao @Inject constructor(context: Context) {

```

Comment:
Same goes for this, I'd suggest putting this class in the same file as StudyBurstActivity

Reply:
Oh I mean in same file as StudyBurstViewModel